src/infrastructure/figma/figma-service.ts

```diff
@@ -30,6 +30,10 @@ const extractDataFromFigmaUrlOrThrow = (url: string): FigmaUrlData => {
 	return urlData;
 };
 
+export const buildIssueTitle = (issueKey: string, issueSummary: string) => {
+	return `[${issueKey}] ${issueSummary}`;
```

Comment:
This is the shape of title that Figma's widget/plugin construct and store, applying it here as well for consistency.